monitor/views.py

```diff
@@ -104,6 +105,21 @@ def queries_stats_ajax(request):
     return JsonResponse(json.load(urllib2.urlopen(req)))
 
 
+@login_required
+def api_usage_stats_ajax(request):
+    query = urllib.urlencode({
+        'query': 'api_client_username:%s' % (request.user.username),
+        'range': '604800',
+        'filter': 'streams:530f2ec5e4b0f124869546d0',
+        'interval': 'day'
+    })
+    req = urllib2.Request("http://mtg-logserver.s.upf.edu/graylog/api/search/universal/relative/histogram?%s" % (query))
```

Comment:
can we use `requests` here, for the same reason in the paypal query stuff - better retry/error catching capabilities, and an API to pass username/passwords